src/css/base.scss

```diff
@@ -1661,6 +1665,10 @@ button.anchor {
     border-bottom: solid 1px var(--N200);
 }
 
+.border-bottom-n1 {
```

Comment:
we already have this class as `border-bottom`

Reply:
it is with different color